Release4.md

```diff
@@ -1,36 +1,123 @@
+We have created a [systematic approach](https://github.com/SovereignCloudStack/docs/blob/main/community/contribute/adding-docs-guide.md) to structure documentation which already has been implemented for the [OpenStack Image Manager](https://docs.scs.community/docs/category/openstack-image-manager/),
+the [OSISM testbed](https://docs.scs.community/docs/category/osism-testbed/) and the [K8s Cluster API Provider](https://docs.scs.community/docs/category/k8s-cluster-api-provider/). More will follow in a continuous manner.
+
+Our community has creates a growing amount of [blog articles](https://scs.community/blog/) which also help to understand the SCS project, its community and the technology that is worked on.
+
 ## Component Versions and User-visible improvements (highlights)
 
+* [OpenStack Zed release](https://releases.openstack.org/zed/highlights.html)
+* Ceph Quincy is available, the default release of Ceph is still Pacific.
+* The base infrastructure is provided by
+  [OSISM 5.0.0](https://release.osism.tech/notes/5.0.0.html)
+  which in turn builds on top of kolla and kolla-ansible.
+* With [Cloud-in-a-Box](https://github.com/osism/cloud-in-a-box) there is an easy way to get SCS up and running on a single hardware node as a test environment. There are two blog posts ([part 1](https://scs.community/2023/03/15/ciab/) and [part2](https://scs.community/2023/03/15/ciab-2/)) covering it.
+* For new deployments of the IaaS reference implementation Ubuntu 22.04 is recommended while existing installations can be upgraded to R4 while staying on Ubuntu 20.04. With Release 5, upgrading to Ubuntu 22.04 will be required.
+* With [osism/node-image](https://github.com/osism/node-image) an iso image for much easier bootstrapping of new OSISM environments is available now
+* The software for our [Kubernetes Cluster-API reference implementation](https://github.com/SovereignCloudStack/k8s-cluster-api-provider) has been updated and highlights are covered in own [release notes](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/blob/main/Release-Notes-R4.md).<!--FIXME: We should link to docs.scs.community here.-->
+
+
 ## New Features (Highlights)
 
 ### Operator focused improvements
 
+* The [Openstack Image Manager](https://github.com/osism/openstack-image-manager) has seen many improvements and is the reference command to assure the images available comply with the [SCS Image Standard]( ) // FIXME LINK
+* For Ceph, special playbooks were added to validate the deployment status of the OSD, MON and MGR services in OSISM. The commands for use are `osism validate ceph-osds`, `osism validate ceph-mons`, and `osism validate ceph-mgrs`.
+* The [testbed](https://github.com/osism/testbed) uses per default a proxy for container pulling. This will allow for airgapped installations out of the box. Please note: a full airgap support (with local mirrors, etc.) will follow in a future release.
```

Comment:
```suggestion
* The [testbed](https://github.com/osism/testbed) uses per default a proxy for container pulling. This will allow for airgapped installations out of the box.
```

I'd like to refrain from making forward promises like these. We can cover that in a blogpost (just like we did for R4). Opinions? @jschoone @garloff @itrich @maxwolfs

Reply:
We could also say that "More work on this will follow." just to indicate that is incomplete and we continue to work on making it more complete. This is about expectation setting. "Yes, we have something and we believe it's useful enough already to talk about it, but we are fully aware that more work is needed and plan for it."

You are right of course: The release notes are not the place to make a lot of forward-looking statements.